Simplenote/Classes/SPNoteEditorViewController.m

```diff
@@ -322,11 +280,12 @@ - (void)highlightSearchResultsIfNeeded
             
             NSString *searchDetailFormat = count == 1 ? NSLocalizedString(@"%d Result", @"Number of found search results") : NSLocalizedString(@"%d Results", @"Number of found search results");
             self->searchDetailLabel.text = [NSString stringWithFormat:searchDetailFormat, count];
-            
+            self->searchDetailLabel.alpha = UIKitConstants.alpha0_0;
+
             [UIView animateWithDuration:0.3
                              animations:^{
                                  
-                                 self->searchDetailLabel.alpha = 1.0;
+                                 self->searchDetailLabel.alpha = UIKitConstants.alpha1_0;
```

Comment:
Did you mean to access its ivar here? If not can you use dot notation instead?

Reply:
This one doesn't have a property (yet), **lots** of refactors on the way. Apologies about the ugly (temporary!!) snippet!!!